src/wasm-parser.ts

```diff
@@ -49,7 +50,7 @@ const __vite__wasmModule = await ${names.initWasm}({ ${imports
       ({ from, names }, i) =>
         `${JSON.stringify(from)}: { ${names.map((name, j) => `${name}: __vite__wasmImport_${i}_${j}`).join(", ")} }`
     )
-    .join(", ")} }, ${names.wasmUrl});
+    .join(", ")} }, ${names.wasmUrl}, ${JSON.stringify(rootPath)});
```

Comment:
Will the injection of `rootPath` leak the project dir in a production build for browser?